### PR TITLE
Add nudge for deprecated functions.

### PR DESCRIPTION
**Pull request by BlakeMScurr**

### CODE REVIEW

File: registry/tenets.yaml [hunk 1]
```diff
@@ -119,6 +119,8 @@ codelingo:
         hasIssues: false
       break-select-in-for:
         hasIssues: false
+      deprecated:
+        hasIssues: false
```
Comment (Daanikus) [inline registry/tenets.yaml]: :tada:

File: tenets/codelingo/go/deprecated/codelingo.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+tenets:
+  - name: deprecated
+    flows:
```
Comment (Daanikus) [inline tenets/codelingo/go/deprecated/codelingo.yaml]: s/flows/actions

Comment (Daanikus) [reply tenets/codelingo/go/deprecated/codelingo.yaml]: Does lingo init still generate with "flows"?

Comment (BlakeMScurr) [reply tenets/codelingo/go/deprecated/codelingo.yaml]: Ooh, yeah it does. And it seems to work too 🤔

Comment (Daanikus) [reply tenets/codelingo/go/deprecated/codelingo.yaml]: We support both, so that's good!

**Pull request merged**